src/components/CustomMUIComponents/CustomButtons.js

```diff
@@ -17,10 +17,11 @@ export const PrimaryButton = ({content, width, data_test}) => {
 }
 
 
-export const PrimaryButton2 = ({content, width, onClick, colour, disable, data_test}) => {
+export const PrimaryButton2 = ({content, width, onClick, colour, disable, data_test, value}) => {
     return (
         <Button
             data-test={data_test}
+            value={value}
```

Comment:
Does adding `value` parameter to this button not break other buttons? Can you not just read the `content` parameter instead of adding a new `value` parameter?

Reply:
all the other Primary buttons 2 that im using in that page and the other pages are working fine, i thought this was a optional param that you can choose to fill or not

Reply:
Hmm, yeah you're probably right about that. As long as it's tested and doesn't break anything else, it should be fine. If you can get the value of `content` instead (since they both have the same value), then that would be preferable to avoid unnecessary redundancy.